client/src/ui/js/custom.js

```diff
@@ -0,0 +1,201 @@
+let timer;
+
+/**
+ * Decode the String so that special characters will be automatically removed.
+ *
+ * @param html The string with special characters.
+ * @returns {string} Decoded string.
+ */
+function decodeHtml(html) {
+
+    let txt = document.createElement("textarea");
+    txt.innerHTML = html;
+    return txt.value;
+}
+
+/**
+ * Function to handle specific button clicked on diagram.
+ *
+ * @param elem The specific element identifier.
+ * @param title The title to show in the Modal view.
+ * @param req The request variable.
+ * @param res The response variable.
+ * @param type The type of the variable.
+ */
+function buttonClick(elem, title, req, res, type) {
+
+    $(".block").on("click", elem, function () {
+        showModal(title, req, res, type);
+    });
+}
+
+/**
+ * Function to show the modal with specific content manipulation.
+ *
+ * @param title The title to show in the Modal view.
+ * @param req The request variable.
+ * @param res The response variable.
+ * @param type The type of the variable.
+ */
+function showModal(title, req, res, type) {
+
+    const XML = "xml";
+    const JSON = "json";
+    clearInterval(timer);
+    $('#modal-title').text(title);
+
+    if (type === XML) {
+        $("#simpleUseCase").simpleXML({
+            xmlString: decodeHtml(req),
+            collapsedText: "...",
+        });
+    } else if (type === JSON) {
+        $('#myTab').show();
+        const req_elem = '#req-tab-val';
+        const req_elem_menu = '#req-tab-menu';
+        const res_elem = '#res-tab-val';
+        const res_elem_menu = '#res-tab-menu';
+
+        $(req_elem).empty();
+        $(res_elem).empty();
+        $(req_elem_menu).hide();
+        $(res_elem_menu).hide();
+        if (req) {
```

Comment:
Keep a new line before the `if` statements

Reply:
Fixed with https://github.com/wso2-extensions/identity-tools-plugin-vscode/pull/7